examples/connext_dds/remote_procedure_call/c++11/Inventory_service.cxx

```diff
@@ -81,31 +81,17 @@ class InventoryImpl : public InventoryService {
     std::mutex mutex;
 };
 
```

Comment:
I created this function because we were creating the participnat in the same scope where we were calling `dds::domain::DomainParticipant::finalize_participant_factory();`. So the participant destructor was not invoked by the time the factory was finalized. producing a precondition error